sale_order_import/wizard/sale_order_import_view.xml

```diff
@@ -5,17 +5,16 @@
 -->
 
 <openerp>
-<data>
 
 <record id="sale_order_import_form" model="ir.ui.view">
     <field name="name">sale.order.import.form</field>
     <field name="model">sale.order.import</field>
     <field name="arch"  type="xml">
         <form string="Import Sale Orders">
-            <group name="technical" invisible="1">
+            <group string="technical" invisible="1">
```

Comment:
@Kiplangatdan Why was it necessary to change all the `name` occurrences to `string` or `group` ? IMO `name` is the preferred way to make the object accessible to inheriting views, and it's commonly used in Odoo 9.

If the aim was to add a caption, you can leave the `name` and add `string` separately.